gptme/tools/base.py

```diff
@@ -90,42 +133,13 @@ def __eq__(self, other):
             return False
         return self.name == other.name
 
+    def is_executable(self):
+        return bool(self.execute)
+
 
 @dataclass(frozen=True)
-class ToolUse:
+class BaseToolUse:
     tool: str
-    args: list[str]
-    content: str
-    start: int | None = None
```

Comment:
I really don't think we need to subclass for this.

Reply:
Removed.